plugins/fuse/server/server_fuse.py

```diff
@@ -375,29 +375,37 @@ def destroy(self, path):
 @atexit.register
 def unmountAll():
     """
-    Unmount all mounted FUSE mounts.
+    Unmount all mounted FUSE mounts.  Ignore failues to unmount.
     """
     for name in list(_fuseMounts.keys()):
-        unmountServerFuse(name)
+        try:
+            unmountServerFuse(name)
+        except GirderException:
+            pass
 
 
 def unmountServerFuse(name):
     """
-    Unmount a mounted FUSE mount.  This may fail if there are open files on the
-    mount.
+    Unmount a mounted FUSE mount.  This may fail and raise an exception if
+    there are open files on the mount.
 
     :param name: a key within the list of known mounts.
     """
     with _fuseMountsLock:
-        entry = _fuseMounts.pop(name, None)
+        entry = _fuseMounts.get(name)
         if entry:
             events.trigger('server_fuse.unmount', {'name': name})
             path = entry['path']
             # Girder uses shutilwhich on Python < 3
             if shutil.which('fusermount'):
-                subprocess.call(['fusermount', '-u', os.path.realpath(path)])
+                result = subprocess.call(['fusermount', '-u', os.path.realpath(path)])
```

Comment:
As a minor nit, [`subprocess.check_call`](https://docs.python.org/2/library/subprocess.html#subprocess.check_call) may be a slightly more straightforward way of doing this, since we'd just catch and re-raise its exception as another type, instead of having the "logic" of figuring out whether to raise one ourselves.

Reply:
A valid point.  I expect that this will change when we switch to "girder mount" instead of a plugin, so I'll leave it for now.